packages/flutter/test/material/input_decorator_test.dart

```diff
@@ -387,6 +387,244 @@ void main() {
     }
   });
 
+  testWidgets('InputDecorator input/label text layout with floatingLabelGap', (WidgetTester tester) async {
+    // The label appears above the input text
```

Comment:
Nit: Period at the end here and elsewhere.

Reply:
done.